gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageFileSystem.java

```diff
@@ -82,7 +83,7 @@
 public class GoogleCloudStorageFileSystem {
 
   private static final GoogleLogger logger = GoogleLogger.forEnclosingClass();
-
+  public static TrackingHttpRequestInitializerFromGcsio httpReq=null;
```

Comment:
We should not use this class in the production code, only in tests.

Reply:
Why do we need this var? I don't see any usages. 

+1 We should avoid using this in prod code